be/src/olap/hll.h

```diff
@@ -124,33 +125,36 @@ class HyperLogLog {
 
     // Check if input slice is a valid serialized binary of HyperLogLog.
     // This function only check the encoded type in slice, whose complex
-    // function is O(1).
+    // function is O(1). 
     static bool is_valid(const Slice& slice);
 
     // only for debug
     std::string to_string() {
         switch (_type) {
-        case HLL_DATA_EMPTY:
-            return {};
-        case HLL_DATA_EXPLICIT:
-        case HLL_DATA_SPARSE:
-        case HLL_DATA_FULL: {
-            std::string str{"hash set size: "};
-            str.append(std::to_string(_hash_set.size()));
-            str.append("\ncardinality:\t");
-            str.append(std::to_string(estimate_cardinality()));
-            str.append("\ntype:\t");
-            str.append(std::to_string(_type));
-            return str;
-        }
-        default:
-            return {};
+            case HLL_DATA_EMPTY:
```

Comment:
case statement does not need to be indented

Reply:
ok